bylaws.md

```diff
@@ -90,11 +117,11 @@ Foreningens bestyrelsen kan fastsætte regler der lader foreningens medlemmer ud
 
 <!-- [en] -->
 # Dues, finance og sponsorater
-Foreningens medlemmer betaler kontigent til ACM og der opkræves intet selvstændigt kontigent til foreningen.
+The Chapters fiscal year begins on July 1st and closes June 30th.
```

Comment:
Requirement of ACM that an annual financial report is sent back to the mother ship.